src/carbon-calculation/interfaces/EC3Configurations.ts

```diff
@@ -223,5 +223,5 @@ export interface EC3ConfigurationMaterial {
    * @type {string}
    * @memberof EC3ConfigurationMaterial
    */
-  nameColumn: string;
+  nameColumn?: string;
```

Comment:
When can the `nameColumn` be null / undefined?

Reply:
In the widget when a label is created there is a new material and it has a single undefined ```nameColumn```. Perhaps this could be fixed in the widget itself

Reply:
It *should* be fixed in the widget.